### ISSUE TITLE
Extra feature support of Batch Norm layer for Keras (and Tensorflow)

### ISSUE THREAD

**Issue by ZongHong-Lyu** [DigitalMediaProfessionals/tool#14]
The original paper of Batch Norm layer proposed that the operation should be applied per batch for fully connected layers, and be applied per 'global batch' for convolution layers.
(Global batch treats batch size as [b x h x w] if the output size is (b, h, w, c). So the mean is calculated per channel using all feature maps in the batch.)

Current tool's and Caffe's implementation follow this approach. But in Keras and Tensorflow, it is possible to apply the batch normalization along arbitrary axis (or more than one axes). In this case the tool can't convert the network correctly.

I am not sure if there will be networks use this feature in Keras (or Tensorflow). So I assume the priority of this feature support can be low.
